dot/network/service_test.go

```diff
@@ -83,6 +83,9 @@ func createTestBlockResponseMessage(t *testing.T) *BlockResponseMessage {
 	return msg
 }
 
+//go:generate mockgen -destination=mock_block_state.go -package $GOPACKAGE . BlockState
+//go:generate mockgen -destination=mock_syncer.go -package $GOPACKAGE . Syncer
```

Comment:
Can you perhaps have these with `_test.go` suffixes? Or are these imported by other packages :thinking: